huaweicloud/services/acceptance/cci/resource_huaweicloud_cci_pvc_test.go

```diff
@@ -1,17 +1,18 @@
-package huaweicloud
+package cci
 
 import (
 	"fmt"
 	"testing"
 
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
-
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 
 	"github.com/chnsz/golangsdk/openstack/cci/v1/persistentvolumeclaims"
+
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/cci"
 )
 
 func TestAccCCIPersistentVolumeClaims_basic(t *testing.T) {
```

Comment:
```suggestion
func TestAccPersistentVolumeClaims_basic(t *testing.T) {
```
All package prefix need be removed.

Reply:
fixed.